Forwarded: https://github.com/BuddiesOfBudgie/budgie-desktop/pull/128
Author: David Mohammed <fossfreedom@ubuntu.com>
Last-Update: 2022-05-23
Description: [PATCH 4/6] Bump expected supported mutter versions - GNOME 43 and 42
 The PR to support color-scheme is GNOME 42 only and it isn't necessary to put
 lots of code guards for budgie v10.7 that will not be released for GNOME 41

---
 src/wm/meson.build | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/src/wm/meson.build b/src/wm/meson.build
index 52c06959..3063e3eb 100644
--- a/src/wm/meson.build
+++ b/src/wm/meson.build
@@ -18,16 +18,16 @@ budgie_wm_sources = [
 budgie_wm_status_vala_args = []
 
 dep_graphene = dependency('graphene-gobject-1.0', version: '>= 1.10')
-dep_mutter = dependency('libmutter-10', version: gnome_minimum_version, required: false)
+dep_mutter = dependency('libmutter-11', version: gnome_minimum_version, required: false)
 if dep_mutter.found()
-    #budgie_wm_status_vala_args += ['-D', 'HAVE_MUTTER_9']
-    message('Using libmutter-10 ABI from GNOME 42')
-    vapi_mutter = 'libmutter-10'
+    #budgie_wm_status_vala_args += ['-D', 'HAVE_MUTTER_11']
+    message('Using libmutter-11 ABI from GNOME 43')
+    vapi_mutter = 'libmutter-11'
 else
-    dep_mutter = dependency('libmutter-9', version: gnome_minimum_version)
+    dep_mutter = dependency('libmutter-10', version: gnome_minimum_version)
     if dep_mutter.found()
-        message('Using libmutter-9 ABI from GNOME 41')
-        vapi_mutter = 'libmutter-9'
+        message('Using libmutter-10 ABI from GNOME 42')
+        vapi_mutter = 'libmutter-10'
     endif
 endif
 
-- 
2.34.1

